curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md

```diff
@@ -7,7 +7,8 @@ dashedName: step-119
 
 # --description--
 
-On a new line, add the string ` You attack it with your <weapon>.` to the `text` value, replacing `<weapon>` with the player's current weapon.
+On a new line, add the string ` You attack it with your <weapon>.` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts
+with a space so it will properly display. 
```

Comment:
I don't think this needs to be a on new line here

Reply:
Thank you pointing that out. I couldn't actually tell an entire new line was added unnecessarily in my code editor.